### PR TITLE
Adding reusable parameters example

### PR DESCRIPTION
**Pull request by carvilsi**

### CODE REVIEW

File: generators/templates/src/index.coffee [hunk 1]
```diff
@@ -17,16 +17,21 @@ catch error
...
 )
 
 # Swagger page
 app.get('/+swagger', (req, res) ->
   res.sendFile('swagger.yaml', {root: './static/'})
 )
 
+# echo POST endpoint, just to have an example of reusing params at swagger file
+app.post('/+echo', (req, res) ->
+	res.status(200).send("Hi. #{if req.query.name? then req.query.name else 'provide a name as query param'}")
```
Comment (malamili) [inline generators/templates/src/index.coffee]: Could you instead add this at the top:
```
# Utility
ensureParameters = (req, res, parameters...) ->
  for p in parameters
    if !req.query[p]?
      res.status(400).send("Parameter #{p} required")
      return false
  true
```

then the function content becomes like this:
```
 if ensureParameters(req, res, 'name')
   res.send(req.query.name)
```
This way we have the ensureParam util also in this template

**Review by malamili — CHANGES_REQUESTED** [review]
Apart from 1 minor comment looks good!

File: generators/templates/src/index.coffee (hunk 1 above)
Comment (carvilsi) [reply generators/templates/src/index.coffee]: 👍

**Review by malamili — APPROVED** [review]
Nice!

**Pull request merged**